### PR TITLE
update authorize script

### PR DESCRIPTION
**Pull request by horis233**

### CODE REVIEW

File: scripts/authorize-namespace.sh [hunk 1]
```diff
@@ -107,23 +106,23 @@ if [ $? -ne 0 ]; then
 fi
 
 if [ "$TARGETNS" == "$TONS" ]; then
-  echo "Namespace and tonamespace canot be the same namespace."
+  echo "Namespace and to namespace canot be the same namespace."
```
Comment (Daniel-Fan) [inline scripts/authorize-namespace.sh:109]: it should be `cannot`

Comment (horis233) [reply scripts/authorize-namespace.sh:109]: Nice catch

Comment (horis233) [reply scripts/authorize-namespace.sh]: @Daniel-Fan updated

**Review by Daniel-Fan — APPROVED** [review]
/lgtm

**Pull request merged**